core/src/main/scala/com/softwaremill/session/SessionManager.scala

```diff
@@ -84,7 +84,19 @@ trait CsrfManager[T] {
   def config: SessionConfig
 
   def tokenInvalidRejection = AuthorizationFailedRejection
-  def createToken(): String = SessionUtil.randomString(64)
+  def createToken(): String = {
+    val timestamp = System.currentTimeMillis().toString
```

Comment:
the othre managers have an overridable `def nowMillis: Long` (for testing). Would be good to do the same thing here as well

Reply:
Thanks for this suggestion, changed as suggested.